### PR TITLE
Oracle support for kava 13

### PR DESCRIPTION
**Pull request by DracoLi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +41,7 @@
     "typescript": "^4.9.3"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx}": "eslint --cache --fix --max-warnings=0",
+    "*.{js,jsx,ts,tsx}": "eslint --cache --fix",
```
Comment (galxy25) [inline package.json:44]: nit: is it worth it to add explicit ignores for the lint rules we don't care about versus changing the global default to allow warnings?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kava-labs/javascript-sdk",
-  "version": "9.0.3-beta-3",
+  "version": "9.0.3-beta-4",
```
Comment (galxy25) [inline package.json:3]: maybe for version 100 we can shed the beta tag 😅

File: src/client/index.ts [hunk 3]
```diff
@@ -484,19 +528,24 @@ export class KavaClient {
   async postPrice(
     marketID: string,
     price: string,
-    expiry: string,
+    expiry: Date,
```
Comment (galxy25) [inline src/client/index.ts:531]: type tightening for the win 👏🏽

**Review by galxy25 — APPROVED** [review]
lgtm

**Review by rhuairahrighairidh — APPROVED** [review]

**Pull request merged**